plopfile.js

```diff
@@ -67,3 +65,37 @@ module.exports = function (plop) {
     ],
   });
 };
+
+/**
+ * Extract an exported name from an export statement.
+ *
+ * @example
+ * getNameFromExport('export { default as FooBear } from "./FooBear";');
+ * // => 'FooBear'
+ *
+ * @example
+ * getNameFromExport('export type { FooBat } from "./FooBat";');
+ * // => 'FooBat'
+ */
+function getNameFromExport(exportLine) {
+  // Get the exported name. We have to handle cases like `export { default as Foo }` and
+  // `export type { Foo }`. To match these cases, this is currently looking for:
+  //
+  // 1. Opening curly bracket, for either `export {` or `export type {`.
+  // 2. Any characters immediately after the opening bracket. Mostly to match `default as`.
+  // 3. Capture the name of the thing that's exported.
+  // 4. Closing curly bracket.
+  //
+  //                            1 2   3    4
+  //                            ┃ ┃ ┏━┻━┓  ┃
```

Comment:
😮  how did you do that

Reply:
Unicode box drawing characters: https://en.wikipedia.org/wiki/Box-drawing_character